UefiCpuPkg/Library/CpuExceptionHandlerLib/SecPeiCpuExceptionHandlerLibMu.inf

```diff
@@ -0,0 +1,58 @@
+## @file
+#  CPU Exception Handler library instance for SEC/PEI modules.
+#
+#  Copyright (c) 2012 - 2018, Intel Corporation. All rights reserved.
+#  Copyright (c) Microsoft Corporation.
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = SecPeiCpuExceptionHandlerLibMu

```

Comment:
We should look at merging this with the edk2 version of this library. Can you diff them and merge if not radically different? If they are radically different, we should rename this lib to something more meaningful as we look towards upstreaming.

Reply:
They aren't that different so if that's the path we want to take I can do that easily.

Reply:
Let's combine them then so we can upstream this

Reply:
I haven't diffed them, but keep in mind even if the differences are small, check that the intention of a different instance is not needed. You may still be able to merge it together but done with a common code together and separate files for the delta.

Reply:
Closing this thread since we discussed reasons for converging the instances offline.

After looking at the diff, it can relatively easily follow the normal convention of extending code with a library class that a platform can control, providing backward compatible behavior with the NULL instance.
- Better keeps the file in sync with the upstream and makes the changes more visible to reconcile with the upstream.